pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -2105,6 +2098,18 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 
 	setIOThreads(vmi, domain, vcpus)
 
+	builder := NewDomainBuilder(
+		metadata.DomainConfigurator{},
+		network.NewDomainConfigurator(
+			network.WithDomainAttachmentByInterfaceName(c.DomainAttachmentByInterfaceName),
+			network.WithUseLaunchSecuritySEV(c.UseLaunchSecuritySEV),
+			network.WithUseLaunchSecurityPV(c.UseLaunchSecurityPV),
+		),
```

Comment:
It is important to note that this is a temporary solution. We will not want for the infra to expose these here. At the moment this is an intermediate state.

In a follow up, the dependency needs to reverse.

Reply:
As written [here](https://github.com/kubevirt/kubevirt/pull/15986#discussion_r2484808563), we can add the `converter.ConverterContext` as a parameter to the `configurator.Configure` method.
I prefer doing so in a follow up, since it will require moving converter.ConverterContext to a dedicated package in order to avoid a cyclic dependancy.